server/routes/appointments/attendance/handlers/summaries.ts

```diff
@@ -20,11 +22,25 @@ export default class SummariesRoutes {
       await this.activitiesService.getAppointmentAttendanceSummaries(user.activeCaseLoadId, dateOptionDate, user)
     ).filter(s => !s.isCancelled)
 
+    // Get prisoner details for appointments with a single attendee
+    const prisonerNumbers = summaries.flatMap(r => (r.attendees.length === 1 ? r.attendees[0].prisonerNumber : []))
```

Comment:
Wouldn't this be just one prisonerNumber and not prisonerNumbers?  Or am I misunderstanding.